lang/nl.json

```diff
@@ -0,0 +1,955 @@
+            "quick_count": "{ count, plural, one {# productresultaat} other {# productresultaten} } voor '{ search_query }'",
+            "quick_count_live": "producten gevonden voor",
+            "product_count": "Producten ({count})",
+            "content_count": "Nieuws en informatie ({count})"
+        },
+        "faceted": {
+            "selected": {
+                "title": "Verfijnen op",
+                "rating-label": "Beoordeeld met {rating, plural, one {# Star} other {# Stars}} of hoger",
+                "no-filters": "Geen filters toegepast",
+                "clear-all": "Alles wissen"
+            },
+            "range": {
+                "update": "Bijwerken",
+                "min-placeholder": "Min.",
+                "max-placeholder": "Max."
+            },
+            "rating": {
+                "and-up": "& meer"
+            },
+            "toggleSection": "{title} filtersectie in- of uitschakelen",
+            "clear": "Wissen",
+            "more": "Meer",
+            "show-more": "Meer weergeven",
+            "show-less": "Minder weergeven",
+            "show-filters": "Filters weergeven",
+            "hide-filters": "Filters verbergen",
+            "browse-by": "Zoeken op"
+        },
+        "error": {
+            "empty_field": "Zoekveld mag niet leeg zijn."
+        },
+        "tabs_accesibility_hint": "Gebruik de pijlen naar links en rechts om tussen tabbladen te navigeren."
+    },
+    "page_not_found": {
+        "page_heading": "404-fout: pagina niet gevonden",
+        "message": "De pagina die u zoekt, is verhuisd of bestaat niet meer."
+    },
+    "server_error": {
+        "page_heading": "Er is iets fout gegaan...",
+        "message": "Winkel is momenteel niet beschikbaar, probeer het over enkele minuten opnieuw"
+    },
+    "forbidden": {
+        "page_heading": "Log in om door te gaan",
+        "if_you_were_signed_in": "Als u ingelogd was,",
+        "sign_in": "log dan opnieuw in",
+        "to_continue": "om uw werk in een nieuwe sessie te hervatten."
+    },
+    "sitemap": {
+        "page_title": "Sitemap",
+        "show_all_link": "Alles weergeven"
+    },
+    "text_truncate": {
+        "view_more": "Meer details weergeven",
+        "view_less": "Details verbergen"
+    },
+    "maintenance": {
+        "down": "Offline voor onderhoud"
+    },
+    "admin": {
+        "maintenance_header": "Uw winkel is offline voor onderhoud.",
+        "maintenance_tooltip": "Alleen beheerders kunnen de winkel op dit moment bekijken. Ga naar de instellingenpagina van het configuratiescherm om de onderhoudsmodus uit te schakelen.",
+        "maintenance_showstore_link": "Klik hier om te zien wat uw bezoekers zien.",
+        "prelaunch_header": "Uw winkelontwerp is privé. Deel uw site met voorbeeldcode:",
+        "page_builder_link": "Deze pagina in Page Builder ontwerpen"
+    },
+    "carousel": {
+        "arrow_and_dot_aria_label": "Ga naar dia [NUMBER] van",
```

Comment:
this construction has been affected in all languages. English equivalent is:  `"Go to slide [NUMBER] of" `
While original phrase from EN.json: `"Go to slide [SLIDE_NUMBER] of [SLIDES_QUANTITY]",`